src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyValidators/OrganizationPolicyValidator.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyValidators;
 
-public abstract class OrganizationPolicyValidator(IPolicyRepository policyRepository, IEnumerable<IPolicyRequirementFactory<IPolicyRequirement>> factories) : IPolicyValidator
+public abstract class OrganizationPolicyValidator(IPolicyRepository policyRepository, IEnumerable<IPolicyRequirementFactory<IPolicyRequirement>> factories)
 {
```

Comment:
You can also remove the `IPolicyValidator` abstract declarations from this class as well now that it's no longer implementing `IPolicyValidator` (which I think is right).

Reply:
[Same](https://github.com/bitwarden/server/pull/6253#discussion_r2311041763)